gammapy/spectrum/core.py

```diff
@@ -269,6 +269,19 @@ def to_table(self):
         else:
             meta.update(hduclas2='BKG', )
 
+        # Add general optional keywords if the member exists rather than default value. LBYL approach
```

Comment:
Two comments here:

1) Can you please add the new parameters (tstart, tstop, muoneff, zen_pnt) to the docstring and mark them as optional, like here
https://github.com/gammapy/gammapy/blob/master/gammapy/spectrum/core.py#L219
 Otherwise it's hard to see from the docs than on can pass these parameters. 

2) Other optional parameters are added to the FITS file here
https://github.com/gammapy/gammapy/blob/master/gammapy/spectrum/core.py#L249

Can you make the code consistent somehow (either with my approch using ``getattr`` or using your ``hasattr`` approch,  up to you)